test/annotation_layer_test.css

```diff
@@ -54,6 +55,18 @@
   width: 100%;
 }
 
+.annotationLayer .textWidgetAnnotation input[disabled],
+.annotationLayer .textWidgetAnnotation textarea[disabled] {
+  background: none;
+  border: 1px solid transparent;
+}
+
+.annotationLayer .textWidgetAnnotation textarea {
```

Comment:
Just a reminder: don't forget to update this file as well when addressing the comment about `hover`/`focus` in the ` test/annotation_layer_test.css` file.